libs/wingsdk/src/ex/react-website.md

```diff
@@ -0,0 +1,88 @@
+---
+title: React Website
+id: react-website
+description: A built-in resource for creating deployable websites using the React framework.
+keywords: [Website, React, deployment, build]
+sidebar_position: 1
+---
+
+The `ex.ReactWebsite` resource represents a website, built using React, that can be both hosted in the cloud or to run on a development hot-reload server in the sim target.
+
+## Usage
+
+### React Website
+
+#### Initiation
+
+using the default arguments:
+
+```ts
+bring cloud;
+bring util;
+
+let website = new cloud.Website(projectPath: "./client", isDevRun: util.tryEnv("ENV") == "dev");
+```
+
+or customizing them:
+
+```ts
+bring cloud;
+bring util;
+
+let website = new cloud.Website(
+  projectPath: "./client",
+  isDevRun: util.tryEnv("ENV") == "dev"
```

Comment:
it's a flag indicating that we are running a local development server for react (by running npm start) other than building it (and serving the static files via express/s3), how would you call that?

`NODE_ENV ` is a common env var used in node and react, indicating what environment are we on (production, testing, development, etc...) since it's not a node app, I omitted the `NODE` and left with `ENV` should I call it differently?

Reply:
but we don't really use nodeJS 🤔

Reply:
because the sim got 2 modes- development and build... (at the end I just passed a boolean in my examples I think that's ok )